tests/jballerina-unit-test/src/test/resources/test-src/query/simple-query-with-defined-type.bal

```diff
@@ -344,7 +344,7 @@ public function testQueryWithStream() returns boolean {
     NumberGenerator numGen = new;
     var numberStream = new stream<int, error?>(numGen);
 
-    int[]|error? oddNumberList = from int num in numberStream
+    int[]|error oddNumberList = from int num in numberStream
```

Comment:
Shall we add few more tests that we use query-exp to loop through a stream. Try out different completion types of the stream. eg: stream<Type, error>, stream<Type>, stream<Type, error?>, stream<Type, Error1>, etc

Reply:
Updated a0d77b8c38b164221098dea81a8dabfacc3ef8e6